internal/testhelpers/installation/testcmd/main.go

```diff
@@ -0,0 +1,63 @@
+package main
```

Comment:
I'm not a fan of these, this is too much to facilitate a unit test. Just cover it with an integration test, or test the units you're actually interested in.

Reply:
I think I can now remove these test scripts and instead move the test cases to the `update_int_test.go` file, but I will do that in #1311, as they depend on functioning update files.

Reply:
These tests were super helpful though, to stay confident about what I needed without having to have a full update yet...  And it will be easy to transfer them now...